CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+1.0.0
```

Comment:
I deleted this file since we rely on the releases for versioning and I didn't see this file in other sso repos.